.github/workflows/dev_module_build-on-self-hosted-runner.yml

```
@@ -55,18 +55,34 @@ jobs:
     needs: check_label
     outputs:
       modules_module_tag: ${{ steps.modules_module_tag.outputs.MODULES_MODULE_TAG }}
+      edition: ${{ steps.modules_module_tag.outputs.MODULE_EDITION }}
     steps:
       - name: Set vars
         id: modules_module_tag
         run: |
           if [[ "${{ github.ref_name }}" == 'pre-alpha' || "${{ github.ref_name }}" == 'main' ]]; then
             MODULES_MODULE_TAG="${{ github.ref_name }}"
-          else
+          elif [[ -n "${{ github.event.pull_request.number }}" ]]; then
             MODULES_MODULE_TAG="pr${{ github.event.pull_request.number }}"
+          elif [[ -n "${{ github.event.inputs.pr_number }}" ]]; then
+            MODULES_MODULE_TAG="pr${{ github.event.inputs.pr_number }}"
+          else
+            echo "Cannot set MODULES_MODULE_TAG variable"
+            echo "Exit"
```

`::error` will display message in Summary section.

```suggestion
            echo "::error title=Module image tag is required::Can't detect module tag from workflow context. Dev build uses branch name as tag for pre-alpha and main branches, and PR number for builds from pull requests. Check workflow for correctness."
```